common/common.scss

```diff
@@ -56,7 +56,7 @@ tbody {
   + .badge-category-bg {
     width: 10px;
     margin-left: -5px;
-    border: 1px solid white;
+    border: 1px solid var(--secondary);
```

Comment:
Small change for dark themes, subcategories will have a dark (instead of white) circle around the category bullet.